bootstrap/kubeadm/internal/locking/control_plane_init_mutex_test.go

```diff
@@ -305,6 +395,18 @@ func (l *logtests) Info(level int, msg string, keysAndValues ...interface{}) {
 		data: data,
 	})
 }
+
+func (l *logtests) Error(err error, msg string, keysAndValues ...interface{}) {
```

Comment:
Had to add a function to the log test as it was giving a null pointer when calling Error.